response -> 22654
  value is not a valid dict (type=type_error.dict)
response -> 22655
  value is not a valid dict (type=type_error.dict)
response -> 22656
  value is not a valid dict (type=type_error.dict)
response -> 22657
  value is not a valid dict (type=type_error.dict)
response -> 22658
  value is not a valid dict (type=type_error.dict)
response -> 22659
  value is not a valid dict (type=type_error.dict)
response -> 22660
  value is not a valid dict (type=type_error.dict)
response -> 22661
  value is not a valid dict (type=type_error.dict)
response -> 22662
  value is not a valid dict (type=type_error.dict)
response -> 22663
  value is not a valid dict (type=type_error.dict)


파이단틱에서 orm_mode를 깜빡하고 지정을 안하면 위와같은 오류가 발생..
별거 아니지만 맨날 하는 실수여서 적어봄 ;
